documentation/service_admin/installing-kubernetes.adoc

```diff
@@ -88,6 +88,22 @@ kubectl create secret generic keycloak-credentials --from-literal=admin.username
 kubectl apply -f ./kubernetes/addons/standard-authservice.yaml
 ----
 
+. Create keycloak configuration used by controller and service. For the messaging console and
+keycloak controller to work, you need to specify the `httpUrl` setting. To get the service ip:
```

Comment:
Not a blocker, but to me it would be clearer to describe this as being for oauth (if it is not set, the console can still use basic auth). That also makes it clearer why the auth path is there.

Reply:
@grs It is also used by keycloak-controller to connect to keycloak, so not only oauth (which is why the parameter name was changed).